native_client/metadata.h

```diff
@@ -0,0 +1,20 @@
+#ifndef METADATA_H_
+#define METADATA_H_
```

Comment:
Just a quick overview, because it's late and I'm in no shape to correct review, but if those structures are part of the exposed API, then we should have them in `deepspeech.h`, no?

Reply:
Yep, this could go in `deepspeech.h` or at least be included from there.